lib/Raven/Client.php

```diff
@@ -748,6 +759,11 @@ protected function get_http_data()
         // instead of a mapping which goes against the defined Sentry spec
         if (!empty($_POST)) {
             $result['data'] = $_POST;
+        } elseif (isset($_SERVER['CONTENT_TYPE']) && stripos($_SERVER['CONTENT_TYPE'], 'application/json') === 0) {
+            $raw_data = $this->getInputStream() ?: false;
```

Comment:
Since we will never need the input stream for other stuff, maybe we can wrap the Json handling inside that method?

Reply:
You should probably add input stream handling for XML as well. This test is just for JSON. I really didn't want to get into XML parsing so I left that out. We actually have to deal with a legacy XML API for one project. Once I have to debug that I'll make a new patch :)

This could be abstracted into a `parse()` method as it gets bigger. I didn't want to spend too much time on this knowing that 2.0 is on the horizon.